<!DOCTYPE html>
<html lang="">
  
  <head>
  <meta charset="UTF-8">
  <title>Boettiger Lab at Stanford University</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="theme-color" content="#157878">
  <link rel="stylesheet" href="/css/normalize.css">
  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" href="/css/cayman.css">
</head>

  <body id="page-top">
    <section class="page-header">
  <h1 class="project-name">Boettiger Lab at Stanford University</h1>
  <h2 class="project-tagline"></h2>
  <a href="#" class="btn">View on GitHub</a>
  <a href="#" class="btn">Download .zip</a>
  <a href="#" class="btn">Download .tar.gz</a>
</section>
	---
   layout: page
   title: About
   permalink: /about/
   cover_image: https://boettigerlab.github.io/images/lasers2.jpg
---

![lasers](https://boettigerlab.github.io/images/lasers2.jpg)

## Our Interests
We seek to understand the control of gene expression. Differences in gene expression underlie the tremendous variety of cell types in our bodies and account for most of the innate differences between you and me or between me and chimpanzee. These differences are encoded in the non-transcribed parts of our genome called cis regulatory elements, regions that bind proteins (in a sequence dependent manner), which regulate transcription of surrounding genes. Surprisingly, these regulatory elements can be very far away (in linear sequence) from the transcribed elements they control, frequently tens to hundreds of thousands of basepairs apart.  A major direction in the lab is to understand how such long-range interactions occur, how they achieve target specificity, and how they may be reprogrammed by alterations to the genome sequence.

We believe the answers to these questions require understanding the 3-D organization of the genome. While interactions between regulatory elements and genes are long-range, they still occur only on the same chromosome (in cis) and are not known beyond the scale of a couple of megabases, suggesting physical proximity of the elements is necessary for regulation. Moreover, if a regulatory element is artificially placed directly adjacent to a new promoter that it does not normally regulate, it will typically activate transcription from that promoter, indicating that physical proximity is generally sufficient for regulation.  The genome must therefore be folded in such a way to allow communication between all the sequences that need to interact and to segregate those that should not interact.  What this 3-D organization looks like, how it is established, how it changes over development, and what the consequences are for the control of gene expression are all poorly understood questions, which our lab is working to answer.

## Our Tools
To answer these questions we need new tools.  Our lab is engaged in using developing and combining new technologies to enable this research, including:
* super-resolution imaging
* single molecule microscopy
* genetic engineering
* next generation sequencing approaches
* mathematical and biophysical modeling

See our research projects below for some examples of this approach in action.

	---
   layout: page
   title: Research
   permalink: /Research/
   cover_image: https://boettigerlab.github.io/images/lasers2.jpg
---


In order to study the 3D DNA structures which regulate contacts between cis-regulatory elements (CREs) and their target transcribed elements (TEs), we are developing a super-resolution imaging approaches capable of resolving tens to hundreds of individual CREs, TEs, and intervening sequence in single cells of sectioned embryonic tissue and cultured cells at better than 20 nm resolution. We are applying this approach first to a selection of well-studied loci from Drosophila embryos and vertebrate cultured-cells, where the CREs have been mapped and the TEs they regulate identified through prior extensive genetic work. By observing the higher order structure of these loci in both wildtype and mutant conditions and in distinct cell-types that differ in TE expression at the locus, we aim to identify general principles that regulate physical interactions and shape expression. We plan to combine these experiments with simultaneous imaging of nascent RNA and mature transcripts in single cells, to determine which physical structures promote and which impede gene expression. We will also plan to use genetic manipulations and interspecies comparison of CREs to understand the sequence variation within and between species that result in phenotypic variation of key traits such as facial features and disease susceptibility.

    <section class="main-content">
      
      <div id="home">
  <h1>Blog Posts</h1>
  <ul class="posts">
    
      <li><span>12 Feb 2016</span> &raquo; <a href="/main/2016/02/12/sample-content.html">Welcome to Jekyll!</a></li>
    
  </ul>
</div>

      <footer class="site-footer">
  <span class="site-footer-owner"><a href="">Boettiger Lab at Stanford University</a> is maintained by <a href=""></a>.</span>
  <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a>.</span>
</footer>


    </section>

  </body>
</html>
